lib/post/pages/post_page.dart

```diff
@@ -57,6 +57,11 @@ class _PostPageState extends State<PostPage> {
   @override
   void dispose() {
     _scrollController.dispose();
+
+    if (context.read<ThunderBloc>().state.isFabOpen) {
+      context.read<ThunderBloc>().add(const OnFabToggle(false));
```

Comment:
Hmm, this doesn't seem to work. The problem of the FAB staying open when navigating back from a post is now happening again (whether I press the UI back button, the OS back button, or swipe).